src/deepmol/models/deepchem_models.py

```diff
@@ -199,17 +202,42 @@ def predict_on_batch(self, dataset: Dataset) -> np.ndarray:
         """
         return super(DeepChemModel, self).predict(dataset)
 
-    def save(self):
+    def save(self, file_path: str = None):
         """
-        Saves deepchem model to disk using joblib.
+        Saves deepchem model to disk.
+
+        Parameters
+        ----------
+        file_path: str
+            Path to the file where the model will be stored.
         """
-        save_to_disk(self.model, self.get_model_filename(self.model_dir))
+        if file_path is None:
+            raise ValueError("DeepChemModel can not be saved to disk without specifying a file_path.")
+        else:
+            # write self in pickle format
+            if isinstance(self.model, KerasModel):
+                self.model.model.save_weights(os.path.join(file_path, 'model_weights'))
+            elif isinstance(self.model, TorchModel):
+                self.model.save_checkpoint(max_checkpoints_to_keep=1, model_dir=file_path)
 
-    def reload(self):
+    def load(self, file_path: str = None):
```

Comment:
same as in save

Reply:
For deepchem models can we do this: https://github.com/deepchem/deepchem/issues/1603#issuecomment-626354974